fabfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+import yaml
+
+from fabric.api import env, run
+
+
+# Load in server config, here's a quick example:
+#   role_name:
+#     hosts:
+#       - ubuntu@12.34.56.78
+#     broker_url: pyamqp://user:pass@host:port/vhost-gpu
+#     is_gpu: true
+#     docker_image: codalab/competitions-v2-compute-worker:nvidia
+#
+# You select the role to run with like s:
+#   $ fab -R role_name <command>
+env.roledefs = yaml.load(open('server_config.yaml').read())
+
+
+def status():
+    """Gets status of all docker containers on server"""
+    run("docker ps")
```

Comment:
this only gets status of running containers, not all containers. should either add `-a` to the command or update the docstring, depending on what is actually desired.

Reply:
added